GeoLib/Raster.cpp

```diff
@@ -148,9 +148,7 @@ double Raster::interpolateValueAtPoint(MathLib::Point3d const& pnt) const
 	for (unsigned j=0; j<4; ++j)
 	{
 		// check if neighbour pixel is still on the raster, otherwise substitute a no data value
-		if ( (xIdx + x_nb[j]) < 0 ||
-		     (yIdx + y_nb[j]) < 0 ||
```

Comment:
Please lookup the standard for integral conversions.
An int cannot be larger than std::size_t.

To end the discussion on this point here I opened an issue which needs a resolution: https://github.com/ufz/ogs/issues/693